docs/index.rst

```diff
@@ -140,7 +140,7 @@ chance to grab a reference to his original function:
    :linenos:
 
     from myapp import logged_in
-    result = logged_in.original_func(None)
+    result = logged_in.original_function(None)
     self.assertEqual(result['result'], 'Logged in')
 
 That works.  But it's just a little weird.  Since the ``jsonify``
```

Comment:
I also want to add a note about functools.wraps that adds `func.__wrapped__`, which is better than `original_func` as an official convention, but does not remove the point and usefulness of venusian

Reply:
I have some dim memory of intentionally not using `functools.wrap` because it lost information of some kind, but I can't remember what it was.  It may not be applicable these days, but I doubt we want to change things around to be "more correct" at the expense of possibly breaking things.

Reply:
Also, I think in this particular case, the use of manually attaching original_function instead of `functools.wraps` was only so all the code was in the same place for didactic purposes so when someone reads the docs they don't need to know how `functools.wraps` works under the hood.  Venusian apparently still doesn't use `functools.wraps`, and I still can't remember why we don't, but we shouldn't bother in any case.